docs/book/src/getting-started/testdata/project/dist/chart/values.yaml

```diff
@@ -37,6 +37,15 @@ manager:
         cpu: 10m
         memory: 64Mi
 
+  # Pod's affinity
+  affinity: {}
+
+  # Pod's node selector
+  nodeSelector: {}
+
+  # Pod's tolerations
+  tolerations: []
```

Comment:
That is valid only for the mananger pod right?
Could we just add a comment like `Manager Pod's ... ` so that we can clarify it?
Also, could we add examples over how to fill those fields in the docs? 
WDYT?